phpunit.xml

```diff
@@ -1,23 +1,20 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="./vendor/autoload.php" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd">
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" bootstrap="./vendor/autoload.php" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.5/phpunit.xsd" cacheDirectory=".phpunit.cache">
   <php>
     <env name="SYMFONY_DEPRECATIONS_HELPER" value="weak"/>
   </php>
-  <coverage>
+  <testsuites>
+    <testsuite name="Doctrine auditable bundle test suite">
+      <directory suffix="Test.php">./tests</directory>
+    </testsuite>
+  </testsuites>
+  <source>
     <include>
       <directory suffix=".php">./src</directory>
     </include>
     <exclude>
       <directory>./tests</directory>
       <directory>./vendor</directory>
     </exclude>
-  </coverage>
-  <testsuites>
-    <testsuite name="Doctrine auditable bundle test suite">
-      <directory suffix="Test.php">./tests</directory>
-    </testsuite>
-  </testsuites>
-  <listeners>
-    <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener"/>
-  </listeners>
```

Comment:
1) Test results may not be as expected because the XML configuration file did not pass validation:

  Line 12:
  - Element 'listeners': This element is not expected.